desktop/core/src/desktop/api2.py

```diff
@@ -124,6 +124,20 @@ def decorator(*args, **kwargs):
   return decorator
 
 
+def _safe_conf_get(conf_obj):
+  """Return config value if bound, else the declared default (or None).
+
+  Some app configs (e.g., from `filebrowser.conf`) might not be bound when the
+  app is blacklisted via `app_blacklist`. In that case, the imported object is a
+  bare `Config` (no `.get()`), so return its declared default/dynamic default.
+  """
+  getter = getattr(conf_obj, 'get', None)
+  if callable(getter):
+    return getter()
+  # For unbound Config, use declared default or None
+  return getattr(conf_obj, 'default', None)
```

Comment:
Are we sure that there are no side effects from loading the default config for blacklisted apps?

Reply:
I tested the fix manually and every app was opening fine.

Also if we think about a scenario where let's say the app is not blacklisted but we dont configure these value in the .ini files, then also stuff works with default values only. So that way it's not breaking as well.

Its just, now the default value will come for these few configs even when the app is blacklisted, and in the manually testing I couldn't find any regression.